src/components/BaseModal/BaseModal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './BaseModal.css';
+import computeClassNames from '../../utils/compute-class-names';
+
+interface Props {
+  isClosed?: boolean;
```

Comment:
De acuerdo al uso que se le da al estado: 
` const [isClosed, setIsClosed] = useState(false);`
```
    <BaseModal isClosed={isClosed} renderHeader={() => <h1>Header Testing</h1>} onClose={setIsClosed}>
        <p>Modal Content</p>
      </BaseModal>
```

Tendria mas sentido que el nombre de la propiedad sea open o isOpen, dado que comienza en `false` lo que indica que no esta `open`
```suggestion
  isOpen?: boolean;
```